routes/profile.js

```diff
@@ -31,8 +31,16 @@ router.route('/profile')
       });
     }
     var updatedData = {};
-    // TODO: This is certainly not secure:
-    // Type checking and sanitaton
+    // 
+    var goodData = ["bio", "firstname", "lastname", "major", "email", "username"];
```

Comment:
We don't want the user to change their email or username. They are unique. See the `/profile`. The front-end doesn't provide the option either.